systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1010,6 +1010,60 @@ void testManualTriggeringRollingUpdate() {
         return CLIENT.apps().statefulSets().inNamespace(namespace).withName(ssName).get().getMetadata().getAnnotations();
     }
 
+    @Test
+    @Tag(REGRESSION)
+    void testKafkaJBOD() {
+        int diskCountPerReplica = 2;
+        int kafkaReplicas = 2;
+        int diskSizeGi = 10;
+        resources().kafkaJBOD(CLUSTER_NAME, kafkaReplicas, diskCountPerReplica, diskSizeGi, true).done();
+        // kafka cluster already deployed
+
+        ArrayList pvcs = new ArrayList();
+        for (int i = 0; i < kafkaReplicas * diskCountPerReplica; i++) {
+            LOGGER.info("Getting list of all PVCs in namespace");
+            String volumeName = CLIENT.inNamespace(NAMESPACE).persistentVolumeClaims().list().getItems().get(i).getMetadata().getName();
+            pvcs.add(volumeName);
```

Comment:
Can you call `CLIENT.inNamespace(NAMESPACE).persistentVolumeClaims().list().getItems().get(i)` just one time at the beginning of the cycle and set a variable to avoid subsequent calls? The code should be even more readable.